src/PublicApiGeneratorTests/PublicApiGeneratorTests.csproj

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-    <NoWarn>$(NoWarn);CS0067</NoWarn>
-    <LangVersion>latest</LangVersion>
```

Comment:
Would you mind adding a comment what the suppressions mean?

Reply:
done